### PR TITLE
CloudLinux recognition in Red Hat 8

### PR DESCRIPTION
**Pull request by Firaell**
This PR is an attempt to fix # by .

## Checklist

- [ ] based on top of latest source code 
- [ ] changelog entry included 
- [ ] tests pass in CI 
- [ ] git history is clean 
- [ ] git commit messages are [well-written](https://chris.beams.io/posts/git-commit/#seven-rules)

### CODE REVIEW

**Review by ferki — CHANGES_REQUESTED** [review]
Thanks for your contribution!

This PR does not fully follow the project's expectations, so I'd like to take this chance to kindly link our [Contributing guide](https://github.com/RexOps/Rex/blob/master/CONTRIBUTING.md), specifically the section about [GitHub issues and pull requests](https://github.com/RexOps/Rex/blob/master/CONTRIBUTING.md#github-issues-and-pull-requests).

Please follow the pull request template to provide all expected information either by editing the opening post here, or by opening a new pull request instead of this one.

---

I've downloaded a CloudLinux qcow2 image and started up a local VM with it. Rex got it recognized as Red Hat (due to the presence of the `/etc/redhat-release` file).

That means Rex already picks up the correct Rex::Pkg::Redhat module to manage packages, and the correct Rex::Service::Redhat::systemd module to manage services on CloudLinux. `is_redhat()` also recognizes it correctly as a Red Hat variant (returns `1`), so there's no need to add it to the list of Red Hat clones.

In other words, Rex should already support CloudLinux without any further changes :tada:

So I wondered what do we exactly try to solve here? I would like to learn more about the problem we solve or the new capability we need to add. I'd say it's best to do that via a separate issue (for discussion and design of potential implementations).

Apart from that, we'd need to add a changelog entry here, and run the tests. These steps were checked in the otherwise untouched template, so I'll remove the check for now.
